OakCityV2/DetailViewController.m

```diff
@@ -12,60 +12,24 @@
 @implementation DetailViewController
 
 
--(void)viewDidLoad
-{
-    [_titleLabel loadHTMLString:self.ttitle baseURL:nil];
-    [_companyLabel loadHTMLString:self.company baseURL:nil];
-    [_urlLabel loadHTMLString:self.url baseURL:nil];
-    _urlLabel.dataDetectorTypes = UIDataDetectorTypeLink;
-    [_descriptionLabel loadHTMLString:self.desc baseURL:nil];
+-(void)viewDidLoad {
+    self.titleLabel.text = self.ttitle;
+    self.companyLabel.text = self.company;
+    self.urlTextView.text = self.url;
+    [self.descriptionLabel loadHTMLString:self.desc baseURL:nil];
+    self.automaticallyAdjustsScrollViewInsets = NO;
 }
-@end
 
+- (void)configureView:(Job *)theJob atIndexPath:(NSIndexPath *)indexPath {
+    self.ttitle = theJob.title;
+    self.company = theJob.company;
+    if (theJob.url != (NSString *)[NSNull null]) {
+        self.url = theJob.url;
+    }
+    else {
+        self.url = @"No URL Provided";
+    }
+    self.desc = theJob.desc;
+}
```

Comment:
You're not using the indexPath here either, so don't pass it.  Since you're getting rid of the individual properties in favor of a single, Job property, you'll set this here instead.